typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -66,7 +59,8 @@ export async function placeLimitOrderExample() {
     `Setup Tx Link: https://beta.solscan.io/tx/${setupTxId}?cluster=devnet`
   );
 
-  // To place a limit order, you will need base and quote tokens. For this devnet example, we mint the base and quote tokens.
+  // To place a limit order, you will need base and quote tokens. For this devnet example, we mint the base and quote tokens from a token facuet.
+  // To trade on mainnet, please supply your own base and quote tokens.
```

Comment:
"to trade on mainnet, you will need to have base and quote tokens for the given market"